docs/source/visint.rst

```diff
@@ -1,6 +1,33 @@
 Visualization and Interaction
 *****************************
 
+Plotting a 1D Sweep
+==================
+
+A function called ``plot_sweep1d`` has been created to facilitate creating, solving, and plotting the results of a single-variable sweep. Example usage is as follows:
```

Comment:
I was thinking it would link to autosweeping documentation once we have one of those...